engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-base-subscriptions.controller.js

```diff
@@ -32,7 +32,8 @@ angular.module('Bastion.content-hosts').controller('ContentHostBaseSubscriptions
             'id':                       $scope.$stateParams.contentHostId,
             'organization_id':          CurrentOrganization,
             'sort_by':                  'name',
-            'sort_order':               'ASC'
+            'sort_order':               'ASC',
+            'match_system':             true
```

Comment:
Example would be a virtual-only sub being shown for a physical host.

Yes, that param block is sent to both though only the available call uses that param.